reactor-extra/src/test/java/reactor/retry/RepeatTests.java

```diff
@@ -399,7 +399,7 @@ public void functionReuseInParallel() throws Exception {
 			for (int j = 1; j <= range; j++)
 				values[i * range + j - 1] = j;
 		}
-		RetryTestUtils.<Long>testReuseInParallel(2, 20,
+		RetryTestUtils.<Long>testReuseInParallel(2, 19,
```

Comment:
can you revert back to `20` and change the assertion in `RetryTestUtils` instead to be 
```java
	for (Integer count : backoffCounts.values()) {
			//backoff not invoked anymore when maxIteration reached
			assertEquals(repeatCount, count.intValue());
		}
```